app/controllers/tournaments_controller.rb

```diff
@@ -74,9 +106,23 @@ def destroy
 
   private
 
+  def validate_updated_after (updated_after)
+    p updated_after
+    d = Chronic.parse(updated_after)
```

Comment:
I don't think we need a gem to do this. Both because now the site's vulnerable to code execution bugs in the gem and a simple regex to make sure that the input is the right format (just six digits like YYMMDD for example?)  would be just as good?

Reply:
I'm just going to use iso8601 timestamps, they are URL safe and allow for easy validation